### ISSUE TITLE
Add support for DNS

### ISSUE THREAD

**Issue by dldinternet** [fog/fog-digitalocean#19]
DigitalOcean and the API now supports DNS ;)

**Comment by h0lyalg0rithm** [comment]
Hi @dldinternet I would be great if you could add support for DNS, if you need it urgently.
If you need some help do let me know

I might be free this weekend will add it then.

**Comment by dldinternet** [comment]
Hey there ... Thanks for commenting!
 I started working on it and think it is a good thing to get more experience with Fog since I work with different Cloud libraries all the time. 
I will push what I have when I can and you can critique or advise as you see fit any time 
My fork is here: https://github.com/dldinternet/fog-digitalocean

**Comment by vinc** [comment]
Hi @dldinternet what's the status on your PR?

**Comment by dldinternet** [comment]
Hmmm ... I did the implementation but it fell by the wayside b4 I made a PR. I will have to dust it off and see if it is worthy of a PR

**Comment by dldinternet** [comment]
You mean this? https://github.com/fog/fog-digitalocean/pull/21

**Comment by vinc** [comment]
Yes! You did a good job on that, it'd be cool to merge it if it's working.
